src/components/molecules/CarouselNavigation.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div class="arrow-container">
-		<CarouselArrow v-on:traverse="$emit('traverse', arrow.direction)" :arrow="arrow" :key="arrow.id" v-for="arrow in arrows"/>
+		<CarouselArrow v-on:traverse="$emit('traverse', arrow.direction)" :arrow="arrow" :key="index" v-for="(arrow, index) in arrows"/>
```

Comment:
as i wrote before use atoms and molecules  for layout and in **organisms** define how they behave with each other and adding functionality to buttons/slides/etc. on "organism" component since buttons that emit event makes them unusable in other components and it makes no point of making molecules and atoms since they become locked to only 1 compoent